src/main.rs

```diff
@@ -130,60 +135,16 @@ fn run_once(
     Ok(messages)
 }
 
-fn report_once(messages: &[Message]) -> Result<()> {
-    let (fixed, outstanding): (Vec<&Message>, Vec<&Message>) =
-        messages.iter().partition(|message| message.fixed);
-    let num_fixable = outstanding
-        .iter()
-        .filter(|message| message.kind.fixable())
-        .count();
-
-    if !outstanding.is_empty() {
-        for message in &outstanding {
-            println!("{}", message);
-        }
-        println!();
-    }
-
-    if !fixed.is_empty() {
-        println!(
-            "Found {} error(s) ({} fixed).",
-            outstanding.len(),
-            fixed.len()
-        );
-    } else {
-        println!("Found {} error(s).", outstanding.len());
-    }
-
-    if num_fixable > 0 {
-        println!("{num_fixable} potentially fixable with the --fix option.");
-    }
-
-    Ok(())
-}
-
-fn report_continuously(messages: &[Message]) -> Result<()> {
-    tell_user!(
-        "Found {} error(s). Watching for file changes.",
-        messages.len(),
-    );
-
-    if !messages.is_empty() {
-        println!();
-        for message in messages {
-            println!("{}", message);
-        }
-    }
-
-    Ok(())
-}
-
 fn inner_main() -> Result<ExitCode> {
     let cli = Cli::parse();
 
     set_up_logging(cli.verbose)?;
 
-    let mut settings = Settings::from_paths(&cli.files);
+    // TODO(charlie): Can we avoid this cast?
+    let paths: Vec<&Path> = cli.files.iter().map(PathBuf::as_path).collect();
+    let mut settings = Settings::from_paths(paths)?;
```

Comment:
This looks like a merge / rebase issue -- do you mind restoring the `let mut settings = Settings::from_paths(&cli.files);` line, and removing lines 143 - 145?

Reply:
Ah true. I restored it